__tests__/integration/app.test.js

```diff
@@ -0,0 +1,56 @@
+const app = require("../../app");
+const request = require("supertest");
+const fact = require("../../models/fact");
+const api = require("../../routes/api");
+
+describe("Integration/unit testing app.js", function () {
+  describe("GET /", function () {
+    test("GET request responds with a 200 status code and returns HTML", async function (done) {
+      const response = await request(app).get("/");
+      const { statusCode, text } = response;
+      expect(statusCode).toBe(200);
+      expect(text).toContain("<!DOCTYPE html>");
```

Comment:
can you get a bit more specific about what's returned? like maybe find something in the html that's specific to the documentation?

Reply:
I changed this to look for something more specific